pkgs/tools/misc/ipxe/default.nix

```diff
@@ -20,14 +18,16 @@ let
 in
 
 stdenv.mkDerivation {
```

Comment:
Also needs `rec` so that `version` can be used in `fetchFromGitHub`

```suggestion
stdenv.mkDerivation rec {
```

Reply:
done.